app/scripts/directives/educationselector.coffee

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+###*
+ # @ngdoc directive
+ # @name edudashApp.directive:educationSelector
+ # @description
+ # # educationSelector
+###
+angular.module 'edudashApp'
+  .directive 'educationSelector', [
+    '$routeParams', '$location'
```

Comment:
Just put the function, instead of the array form. There's actually a transform step in `grunt build` that transforms it into the array form before minification, so there's no trade-off.